cloud_testing/kotlin_poc/src/main/kotlin/ftl/gc/GcAndroidMatrix.kt

```diff
@@ -1,17 +1,16 @@
 package ftl.gc
 
 import com.google.api.services.testing.model.AndroidMatrix
-import com.google.common.collect.Lists
+import ftl.config.Devices
 
 object GcAndroidMatrix {
 
-    fun build(
```

Comment:
I don't think AndroidMatrix is what we want here. That will give us all permutations of the listed configs.
I think want a com.google.api.services.testing.model.AndroidDeviceList instead

- [Use AndroidDeviceList for configure the devices]

Reply:
you're right, we'll fix.

![image](https://user-images.githubusercontent.com/1173057/41428051-ec326c4e-6fd6-11e8-9e8f-6311a7b1ffbb.png)